test/com/rjmetrics/sweet_liberty/integration/t_core.clj

```diff
@@ -164,6 +165,15 @@
                        (fn [data result-data ctx]
                          (:post data)))
              (add-post&handler))))
+  (POST "/insert-with-ring-response" []
+        (make-resource
+          (-> (assoc-in default-sweet-lib-config
+                        [:liberator-config :processable?]
+                        (fn [ctx]
+                          (ring-response {:body {:errors "from processable?"}
+                                                 :status  400})))
+              (force-response-through :processable?)
```

Comment:
I think I like yours more because you don't have to put things in two different places for one feature.